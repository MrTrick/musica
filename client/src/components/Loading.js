import React from 'react';
import { withStyles } from '@material-ui/core/styles';

import CircularProgress from '@material-ui/core/CircularProgress';

const styles = (theme) => ({
  root: {
    display: 'flex',
    justifyContent: 'center',
    padding: '2em'
  },
});

function Loading(props) {
  const {classes} = props;

  return (<div className={classes.root}>
    <CircularProgress size={80} />
  </div>);
}

export default withStyles(styles)(Loading);
