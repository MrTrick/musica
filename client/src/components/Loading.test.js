import React from "react";
import ReactDOM from "react-dom";

import Loading from "./Loading";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Loading />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("looks like a loading element should", () => {
  const wrapper = shallow(
    <Loading />
  );
  expect(wrapper).toMatchSnapshot();
});
