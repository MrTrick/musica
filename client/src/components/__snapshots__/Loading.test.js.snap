// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`looks like a loading element should 1`] = `
<Loading
  classes={
    Object {
      "root": "Loading-root-12",
    }
  }
/>
`;
